integration_tests/pages/assess/sufficientInformationConfirmPage.ts

```diff
@@ -0,0 +1,23 @@
+import type { ApprovedPremisesAssessment as Assessment } from '@approved-premises/api'
+import type { YesOrNo } from '@approved-premises/ui'
+
+import AssessPage from './assessPage'
+
+import SufficientInformationConfirm from '../../../server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationConfirm'
+
+export default class SufficientInformationConfirmPage extends AssessPage {
+  pageClass: SufficientInformationConfirm
+
+  constructor(assessment: Assessment, answer: YesOrNo = 'yes') {
+    super('Suitability Assessment', assessment, 'sufficient-information', 'sufficient-information-confirm', '')
+    this.pageClass = new SufficientInformationConfirm({ confirm: answer }, assessment)
+  }
+
+  completeForm() {
+    this.checkRadioByNameAndValue('confirm', this.pageClass.body.confirm)
+  }
+
+  clickSubmit() {
```

Comment:
Do we need this override? It looks like it's doing the same thing as on the parent class https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/integration_tests/pages/page.ts#L158